Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2685,13 +2685,13 @@ ExceptionOr<RefPtr<WindowProxy>> LocalDOMWindow::open(LocalDOMWindow& activeWind
 
     RefPtr localFrame = dynamicDowncast<LocalFrame>(firstFrame->mainFrame());
 
-    // FIXME: <rdar://118280717> Make WKContentRuleLists apply in this case.
-    RefPtr mainFrameDocument = localFrame ? localFrame->document() : nullptr;
-    RefPtr mainFrameDocumentLoader = mainFrameDocument ? mainFrameDocument->loader() : nullptr;
-    if (firstFrameDocument && page && mainFrameDocumentLoader) {
-        auto results = page->protectedUserContentProvider()->processContentRuleListsForLoad(*page, firstFrameDocument->completeURL(urlString), ContentExtensions::ResourceType::Popup, *mainFrameDocumentLoader);
-        if (results.shouldBlock())
-            return RefPtr<WindowProxy> { nullptr };
+    if (firstFrameDocument && page) {
+        RefPtr firstFrameDocumentLoader = firstFrameDocument->loader();
+        if (firstFrameDocumentLoader) {
```

Comment:
nit: these can be in the same line:
if (RefPtr firstFrameDocumentLoader = firstFrameDocument->loader())
Not important, just a little more concise.